package.json

```diff
@@ -25,7 +25,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.15.1",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "next": "9.5.5",
+    "next": "^11.0.0",
```

Comment:
Don't change this. Revert to older change.

Reply:
Yes pls, this has caused many issues before.